frontend/src/components/adminDetails/jobApproval/JobApproval.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import style from './jobApproval.module.css';
+import ApprovalCard from '../approvalCard/ApprovalCard';
+
+interface JobsProps {
+  jobs: any[];
```

Comment:
you know the type, its the Job interface from `jobsToApprove` you can export it and use it here as `jobs: Job[]`

Reply:
On it :grin: